pkg/apis/validation/validation_test.go

```diff
@@ -206,6 +208,18 @@ func Test_ValidateAttributes(t *testing.T) {
 				field.Invalid(field.NewPath("volumeAttributes", "csi.cert-manager.io/privatekey-file"), "/foobar", "filename must not include '/'"),
 			},
 		},
+		"correct literal-subject should not error": {
+			attr: map[string]string{
+				csiapi.IssuerNameKey:     "test-issuer",
+				csiapi.LiteralSubjectKey: literalSubject,
+				csiapi.CAFileKey:         "ca.crt",
+				csiapi.CertFileKey:       "crt.tls",
+				csiapi.KeyFileKey:        "key.tls",
+				csiapi.DNSNamesKey:       "foo.bar.com",
+				csiapi.KeyEncodingKey:    "PKCS8",
+			},
+			expErr: nil,
```

Comment:
suggestion: this will need many more test cases IMO, to ensure that the splitting functionality works as expected!

Reply:
@SgtCoDFish If I may push back, I felt like if I added too many permutations, I'd essentially be testing `github.com/cert-manager/cert-manager/pkg/util/pki.ParseSubjectStringToRawDerBytes`, because that's doing the heavy lifting in this specific case. That said, if you think there is a branch of this logic that could use a better test, I'm open to anything.